versioned_docs/version-8.7/apis-tools/migration-manuals/migrate-to-camunda-user-tasks.md

```diff
@@ -277,11 +273,7 @@ The following table outlines the respective endpoints. Click the endpoints to fo
                 <code>GET /forms/:formId</code>
             </a>
         </td>
-        <td>
-            <a href="../../camunda-api-rest/specifications/get-user-task-form">
-                <code>GET /user-tasks/:userTaskKey/form</code>
-            </a>
-        </td>
+        <td style={{color: "gray"}}>-</td>
```

Comment:
Unless someone can see an 8.7 endpoint that corresponds to this? I don't see one.

Reply:
Does not exist in 8.7, no.